src/coreclr/nativeaot/Runtime/threadstore.inl

```diff
@@ -4,8 +4,15 @@
 #ifdef _MSC_VER
 // a workaround to prevent tls_CurrentThread from becoming dynamically checked/initialized.
 EXTERN_C __declspec(selectany) __declspec(thread) ThreadBuffer tls_CurrentThread;
+
+// the root of inlined threadstatics storage
+// there is only one now.
+// eventually this will be emitted by ILC and we may have as many such variables
```

Comment:
Nit: I do not think we would want to have one of these for each static field. It would scale poorly.

Reply:
I will remove this comment. The design allows for more than one threadstatic base, if needed.